screens/ProductsScreen.tsx

```diff
@@ -84,7 +94,21 @@ export default function ProductsScreen({navigation}: any) {
   );
 }
 
-const CardView = ({imageSource, title, description, price, onPress}: any) => {
+interface CardViewProps {
+  imageSource: string;
+  title: string;
+  description: string;
+  price: number;
+  onPress: (event: GestureResponderEvent) => void;
+}
```

Comment:
Do not use `any`, create a new type and use it for your object (props or anything else).